PHP/PHP Source.sublime-syntax

```diff
@@ -18,10 +18,11 @@ contexts:
 
   statements:
     - include: class
+    - include: enum
     - include: use-statement
     - match: '(?i)^\s*(interface)\s+({{identifier}})\s*'
       captures:
-        1: storage.type.interface.php
+        1: storage.type.interface.php keyword.declaration.interface.php
```

Comment:
ST4 uses a forwared fill for `keyword.declaration`. Hence you don't need to stack it onto `storage.type`. So you can remove it.

Reply:
Is that done invisibly? I can't see it 
![image](https://user-images.githubusercontent.com/6594915/118745931-d811af00-b889-11eb-8a1d-1b314ea68cc3.png)

Reply:
Okay. I checked JS' syntax definition. I guess https://github.com/sublimehq/Packages/pull/2822/commits/77a34829326adfd2f8c4647746957e716877bd27 should resolve this.